egs2/librimix/sot_asr1/conf/tuning/train_pit_asr_transformer_wavlm.yaml

```diff
@@ -0,0 +1,90 @@
+# network architecture
+# wavlm
+# unused_parameters: true
+freeze_param: [
+    "frontend.upstream"
+]
+
+# network architecture
+frontend: s3prl
+frontend_conf:
+    frontend_conf:
+        upstream: wavlm_local
+        path_or_url: "/home/work_nfs6/pcguo/asr/librimix/hub/wavlm_large.pt"
```

Comment:
This path is for your local experiments.

Reply:
Fixed.